flask_simplelogin/__init__.py

```diff
@@ -129,10 +129,10 @@ def wrap(*args, **kwargs):
 
 
 class Message():
-    def __init__(self, text, category="primary", enabled=True):
+    def __init__(self, text="", category="primary", enabled=True):
```

Comment:
Does it make sense to have a message without a text, or with an empty string?

I don't think so… and if that is the case, we should have `text` as a compulsory argument for a better API : )

Reply:
There's a reason for that. If the user wants to disable a message, it makes more sense to do:

```Message(enabled=False)```
than
```Message(text="",enabled=False)```

In that case, it's not really necessary for text to be compulsory.

Reply:
Got it. But still, this API is not intuitive: users have to create a message just to disable it.

What about something like `SimpleLogin.disable_message(login_success')` for example (and leaving the `Message` objects only for the cases in which there is actually a message)?

I can even think about settings like `SIMPLELOGIN_DISABLE_MESSAGES=login_success,logout` being configured at the envvar level : )

Reply:
Alright! I've had some more time to work on this over Thanksgiving break. I've added a disable_messages() function to SimpleLogin. It's still just setting `Message(enabled=False)`, but it should a little more intuitive this way.